vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/video/ScreenRecorderTestRunListener.kt

```diff
@@ -38,16 +38,16 @@ class ScreenRecorderTestRunListener(
 
     private var hasFailed: Boolean = false
     private var recorder: Thread? = null
-    private var receiver: LineListener? = null
+    private var outputListener: LineListener? = null
```

Comment:
Maybe make sense to declare with  NullOutputListener() for avoiding `!!` check?

Reply:
it's refreshed for every test run